### ISSUE TITLE
ReferenceError: versions is not defined

### ISSUE THREAD

**Issue by LestaD** [GitbookIO/nuts#68]
Steps:

```bash
git clone
nvm use 4.4.4
git checkout 3.1.0
npm install
PORT=23000 GITHUB_REPO=lestad/nonstandard.js GITHUB_TOKEN=df0c8..54e0 npm start
open http://localhost:23000
```

Output in console:

```
> nuts-serve@3.1.0 start /Users/ssova/projects/nuts
> node bin/web.js

Listening at http://:::23000
ReferenceError: versions is not defined
    at /Users/ssova/projects/nuts/lib/nuts.js:166:16
    at _rejected (/Users/ssova/projects/nuts/node_modules/octocat/node_modules/q/q.js:844:24)
    at /Users/ssova/projects/nuts/node_modules/octocat/node_modules/q/q.js:870:30
    at Promise.when (/Users/ssova/projects/nuts/node_modules/octocat/node_modules/q/q.js:1122:31)
    at Promise.promise.promiseDispatch (/Users/ssova/projects/nuts/node_modules/octocat/node_modules/q/q.js:788:41)
    at /Users/ssova/projects/nuts/node_modules/octocat/node_modules/q/q.js:604:44
    at runSingle (/Users/ssova/projects/nuts/node_modules/octocat/node_modules/q/q.js:137:13)
    at flush (/Users/ssova/projects/nuts/node_modules/octocat/node_modules/q/q.js:125:13)
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickCallback (node.js:349:13)
```

> node v4.4.4
> npm v3.8.9
> OS X 10.11.4

**Comment by tofuness** [comment]
See https://nuts.gitbook.com/github.html

**Comment by SamyPesse** [comment]
Fixed by #65
